components/text-utils/text-utils.css

```diff
@@ -0,0 +1,50 @@
+h1,
+.xl-text {
+  font-size: 4.8rem;
+  line-height: 6.4rem;
+}
+h2,
+.large-text {
+  font-size: 3.2rem;
+  line-height: 4rem;
+}
+h3,
```

Comment:
Just a suggestion, for readability, I like to keep one empty line between code blocks  :)